packages/default/scss/slider/_layout.scss

```diff
@@ -256,6 +256,10 @@ $slider-draghandle-active-scale: 1 !default;
         }
     }
 
+    [dir="rtl"].k-slider-vertical .k-slider-track {
```

Comment:
The selector and the applied styles should be different. The Colorpicker in RTL for the Material theme is not looking as desired, please change that

Reply:
OK